app/views/queue_items/index.html.haml

```diff
@@ -0,0 +1,32 @@
+%section.my_queue.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %h2 My Queue
+        %table.table
+          %thead
+            %tr
+              %th(width="10%") List Order
+              %th(width="30%") Video Title
+              %th(width="10%") Play
+              %th(width="20%") Rating
+              %th(width="15%") Genre
+              %th(width="15%") Remove
+          %tbody
+            - @queue_items.each do |queue_item|
+              %tr
+                %td
+                  = queue_item.position 
+                %td
+                  = link_to queue_item.video_title, queue_item.video
+                %td
+                  = button_to "Play", nil, class: "btn btn-default"
+                %td
+                  = queue_item.rating
+                %td
+                  = link_to queue_item.category_name, queue_item.category
+                  -# isn't it better to only go for the category to hit the database only once ?
```

Comment:
You are not hitting database more times. queue_item.category will return category and name will retrieve the name attribute from category object.